docs/.vuepress/config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+    lang: 'en-US',
+    title: 'Plio Docs',
```

Comment:
<img width="393" alt="Screenshot 2021-07-05 at 5 58 29 PM" src="https://user-images.githubusercontent.com/13185699/124471589-aa8db000-ddba-11eb-8ce5-62f0654b2905.png">
Better to just write `Plio` - in all repos - `Docs` word is obvious and hence, not used.